### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by katranci** [jiminny/jiminny-waveform#5]
https://github.com/jiminny/jiminny-waveform/blob/c9264b94b39ea979d70ee45aecd6935e26ec0a92/src/Components/Waveform/Waveforms.jsx#L47

Same as #4. `waveformsWidth` is not a shared state and can be kept as a local state instead.

**Comment by notargetbg** [comment]
I agree.. it was like that at first. But now it feels wrong that I moved it. Plus it is very specific to the `<Waveforms>` ...
